apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/ErrorReporting/ErrorReportingExceptionLogger.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Cloud.ErrorReporting.V1Beta1;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Diagnostics.AspNetCore
+{
+    /// <summary>
+    ///  Google Cloud Error Reporting Exception Logger.
+    /// </summary>
+    /// 
+    /// See <see cref="ErrorReportingExceptionLoggerExtension"/>.
+    /// <example>
+    /// <code>
+    /// public void Configure(IApplicationBuilder app)
+    /// {
+    ///     // Use first to ensure all unhandled exceptions are reported.
+    ///     string projectId = "[Google Cloud Platform project ID]";
+    ///     string serviceName = "[Name of service]";
+    ///     string version = "[Version of service]";
+    ///     app.ReportExceptionsToGoogle(projectId, serviceName, version);
+    ///     ...
+    /// }
+    /// </code>
+    /// </example>
```

Comment:
I think this and the "See ..." text might have to go in the remarks section, although I'm not sure. Maybe it would be best to see how the api docs get generated from this and then adjust if necessary.